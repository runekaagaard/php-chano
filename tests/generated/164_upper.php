<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 'Mixed case input')));
foreach ($items as $i) echo $i->input->upper();
