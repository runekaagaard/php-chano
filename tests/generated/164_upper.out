Fatal error: Call to undefined method DtlIter::upper() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/164_upper.php on line 4